crates/circuit/src/operations.rs

```diff
@@ -602,6 +630,7 @@ impl Operation for StandardGate {
                 )
             }),
             Self::UGate => None,
+            Self::RGate => todo!("Add when we have U3"),
```

Comment:
FWIW, what I did on the other gates that depending on `U3` was to just switch them to use `U` instead for their definitions. The whole `U3`, `U2`, `U1` thing is an artifact of the old IBM basis set and we've just never updated the definitions.

Reply:
I implemented some of `define` for `RGate` in terms of `U`. I think there are four cases for input types and I did two of them. But I noticed that these definitions are not used anywhere at the moment; there is even a comment to this effect. So I don't plan to continue.